library/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneFlightTest.java

```diff
@@ -55,6 +59,8 @@ void testInitializationWhenAttaching() {
 
   void validateSteps(List<Step> steps, boolean isAttaching) {
     assertThat(steps.stream().filter(s -> s instanceof CreateSamResourceStep).count(), equalTo(1L));
+    assertThat(
```

Comment:
I am confused, why is this assertion being added if the testing callsite is disabled?

Reply:
1. I added it when attempting to make the test work
2. If we manage re-enable the test as currently written, then that assertion reflects the expected state
So it doesn't do anything functionally right now, but it does make it more obvious to future us what the intent was.